nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/reduce/bool/IsInf.java

```diff
@@ -66,7 +67,8 @@ public String onnxName() {
 
     @Override
     public String tensorflowName() {
-        return "HasInf";
+        throw new NoOpNameFoundException("No tensorflow name found for " + opName());
+        //return "HasInf";
```

Comment:
Let's remove this (and other reduce bool ops here) and instead put a method with exception in BaseOp - for tensorflowName() and onnxName() - same as DynamicCustomOp

Reply:
Done